tools/api-markdown-documenter/src/test/snapshots/markdown/simple-suite-test/deep-config/test-suite-a/testbetanamespace-namespace/index.md

```diff
@@ -0,0 +1,25 @@
+# TestBetaNamespace
+
+[Packages](/) &gt; [test-suite-a](/test-suite-a/) &gt; [TestBetaNamespace](/test-suite-a/testbetanamespace-namespace/)
+
+A namespace tagged as `@beta`.
+
+**WARNING: This API is provided as a beta preview and may change without notice. Use at your own risk.**
+
+## Signature {#testbetanamespace-signature}
+
+```typescript
+export declare namespace TestBetaNamespace
+```
+
+## Remarks {#testbetanamespace-remarks}
+
+Tests release level inheritance.
+
+## Variables
+
+| Variable | Alerts | Modifiers | Type | Description |
+| --- | --- | --- | --- | --- |
+| [alphaMember](/test-suite-a/testbetanamespace-namespace/alphamember-variable) | `Alpha` | `readonly` |  |  |
+| [betaMember](/test-suite-a/testbetanamespace-namespace/betamember-variable) | `Beta` | `readonly` |  |  |
+| [publicMember](/test-suite-a/testbetanamespace-namespace/publicmember-variable) | `Beta` | `readonly` |  |  |
```

Comment:
Note that the alert column for this member is `Beta` despite the item itself being tagged as `@public`, because its parent namespace is tagged as `@beta`.